### PR TITLE
add simple timeout API

### PR DESCRIPTION
**Pull request by smontanaro**
PR in response to 

https://github.com/alexis-mignon/python-flickr-api/issues/131

### CODE REVIEW

File: flickr_api/method_call.py [hunk 1]
```diff
@@ -48,6 +48,16 @@ def disable_cache():
     CACHE = None
 
 
+TIMEOUT = None
```
Comment (beaufour) [inline flickr_api/method_call.py:51]: Let's set it to default of 10s. It's pretty conservative. If a server hasn't sent anything for 10 seconds, something's quite wrong. You can add a comment with the link to the `requests` documentation as a bonus.

**Review by beaufour — CHANGES_REQUESTED** [review]
Overall, this looks great. Two minor tweaks and then let's get it in.

**Comment by smontanaro** [comment]
I don't know how to tell GitHub that the requested changes have been made, but the PR has been updated.

**Comment by beaufour** [comment]
This looks great. Thanks for the patch @smontanaro !

**Pull request merged**